RELEASE.md

```diff
@@ -11,6 +11,7 @@ Please follow the established format:
 ## Major features and improvements
 
 - Create the toggle-bookmark journey that allows bookmarking runs and displaying them as a separate list. (#689)
+- Auto-update experiment runs list whenever user has a new Kedro run (#703)
```

Comment:
suggestion: 
```suggestion
- Set up web sockets to enable subscription endpoint for auto update of experiment runs list on new Kedro runs (#703)
```

Reply:
I'd really like to leave out implementation details from release note. IMHO user doesn't care whether it uses websocket or anything else.

Reply:
haha yea I kinda knew you would mention this - the main reason I am suggesting this is more for Kedro-viz devs in differentiating this from the previous auto-reload feature, for referencing this release incorporating the actual setup of the web socket mechanism and subscription endpoint. 

We can leave out web socket  but would still be good to mention enabling subscription in this case ( which web sockets is implied I guess)

Reply:
Good point. Updated.